Samples/Samples.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
2.2?

Reply:
For some reason VS complained that I didn't have an SDK that was compatible with 2.2.

2.1 works fine 🤷‍♂️

Reply:
Yeah they added a thing to the SDK that complains when you try to use a release that's newer than the SDK. There's an MSbuild thing to suppress it, but I always ask pranav what it is.